docs/docs/recipes/protect-your-api/spring-boot.mdx

```diff
@@ -0,0 +1,260 @@
+---
+sidebar_label: Spring Boot
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+# Protect your API on Spring Boot
+
+:::note
+
+This tutorial assumes you have created an API Resource `http://localhost:3000/protected` in Admin Console.
```

Comment:
This could be misleading, `http://localhost:3000/` would be more common?

Reply:
good call! reorg the content. PTAL